app/views/admin/AdministerUserModal.coffee

```diff
@@ -320,22 +310,24 @@ module.exports = class AdministerUserModal extends ModalView
       }
     }).then (res) =>
       @administratedTeachers.push(teacher)
+    .catch (jqxhr) =>
+      errorString = "There was an error adding teacher, see the console"
+      @userSaveState = errorString
+      console.error errorString, jqxhr
+      @render()
+    .finally =>
       @updateAdministratedTeachers()
```

Comment:
Nitpick/thought: I feel the `updateAdministratedTeachers` method could be moved to an `afterRender` lifecycle method.
Then we could use `@render` in the finally method, and remove the repeated use of both of them through the file.

Reply:
Agree, but do not want to reset work to clean this up.  Will add a comment instead.